README.md

```diff
@@ -1,266 +1,7 @@
-# Getting Started
+# IronCore Getting Started
 
-IronCore is an SDK and cloud service that keeps application data private and
-secure. Get started with IronCore to grant, monitor and revoke access to
-sensitive data in just a few minutes.
+IronCore is an SDK and cloud service that keeps application data private and secure. Get started with IronCore to grant, monitor, and revoke access to sensitive data in minutes.
 
-### Clone, install, start
+Follow along the `getting-started` interactive tutorial at:
```

Comment:
This sentence doesn't read all that well to me. Maybe just make the entire thing a link and word it like "Follow the tutorial to get started" or something like that.

Reply:
That sounds good to me - I'll update it.